eng/pipelines/common/global-build-job.yml

```diff
@@ -222,10 +224,18 @@ jobs:
       - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-wasmer.cmd $(Build.SourcesDirectory)\wasm-tools
         displayName: Install wasmer
 
-    - ${{ if and(eq(parameters.osGroup, 'windows'), eq(parameters.archType, 'x64')) }}:
-      # Install LLVM for the win-x64 build as it will build the clrjit for browser_wasm cross compilation
-      - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.cmd $(Build.SourcesDirectory)\wasm-tools $(Build.SourcesDirectory) ${{ parameters.buildConfig }}
-        displayName: Install/build LLVM
+    - ${{ if eq(parameters.archType, 'x64') }}:
+      - ${{ if eq(parameters.osGroup, 'windows') }}:
+        - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/set-cmake-path.cmd
+          displayName: Set CMake path
+        - script: pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.ps1 -CI -InstallDir $(Build.SourcesDirectory)/wasm-tools -Configs ${{ parameters.buildConfig }}
+          displayName: Install/build LLVM
+      - ${{ if eq(parameters.osGroup, 'linux') }}:
+        - script: $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-pwsh.sh $(Build.SourcesDirectory)/wasm-tools
+          displayName: Install Powershell 7
+        # Install LLVM for the win-x64 and linux-x64 build as it will build the clrjit for browser_wasm cross compilation
+        - script: $(Build.SourcesDirectory)/wasm-tools/powershell7/pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.ps1 -CI -InstallDir $(Build.SourcesDirectory)/wasm-tools -Configs ${{ parameters.buildConfig }}
```

Comment:
`install-pwsh.sh` sets `PATH`. Does a plain `pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.ps1 ...` not work?

Reply:
I think that will work, yes.  Its not actually exercised with this PR but should be fine.